src/js/mep-player.js

```diff
@@ -249,6 +252,11 @@
 
 		// unique ID
 		t.id = 'mep_' + mejs.mepIndex++;
+ 
```

Comment:
No there's no specific reason; I just tried to move as less of the original code as possible, but it shouldn't impact it at all. I think you are good to do that